packages/manager/modules/pci/src/projects/project/storages/databases/database/users/users.controller.js

```diff
@@ -102,6 +106,26 @@ export default class UsersCtrl {
     return $row.isProcessing() || this.database.isProcessing();
   }
 
+  handleGroupChange(newValue, row) {
+    set(row, 'pending', true);
+    this.DatabaseService.editUser(
+      this.projectId,
+      this.database.engine,
+      this.database.id,
+      row.id,
+      {
+        group: newValue,
+      },
+    )
+      .then(() => {
+        set(row, 'group', newValue);
+        set(row, 'pending', false);
+      })
+      .catch(() => {
+        set(row, 'pending', false);
```

Comment:
Can add `set(row, 'pending', false);` in finally block instead

Reply:
Done: https://github.com/ovh/manager/pull/6399/commits/2896544e3ac01fa237eca693984436db48aaa7e3